### DISCUSSION TITLE
buddhism/2024/level_2/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#95] [General]
# buddhism/2024/level_2/

http://shanenull.com/buddhism/2024/level_2/

**Comment by shane0** [comment]
# LEVEL 2

MEDITATION:
Mindfulness / Compassion Contemplation

DHARMA LECTURE TOPICS:

Compassion & Bodhisattva Practices

• The Bodhisattva Ideal
• Compassion Contemplation
• Six Perfections (Charity, Moral Conduct,Tolerance, Diligence, Meditation, Wisdom)
• Introduction to Mahayana
• Four Immeasurable Minds
• Four Ways of Fellowship
• Four Foundations of Mindfulness
• Samantabhadra's Ten Practices

**Reply by shane0** [reply]
## ch 1

avalokiteshvara 

samantabhadra

…

## ch 2

seven round compassion contemplation
antidote for anger

http://shanenull.com/buddhism/2024/seven/

https://youtu.be/YQzBl5F6Sf8?si=wnpqNEtHIRMw18yz

**Reply by shane0** [reply]
## ch 3 charity

**Reply by shane0** [reply]
http://shanenull.com/buddhism/2024/six_paramitas/

**Reply by shane0** [reply]
#97

**Comment by shane0** [comment]
# sutras

- [sutras](https://www.buddhajewel.org/en/sutras/)
- [platform](https://www.buddhajewel.org/wp-content/uploads/2020/02/platform-sutra-chap-ii-v0.3.8-20131216.pdf)

**Reply by shane0** [reply]
We only have to purify
our mind so that the six consciousnesses exit the six gates
(senses) without being contaminated or defiled by the six
dusts (sense objects).

**Reply by shane0** [reply]
One who has mastery of the mind
And mastery in teaching the Dharma
Is like the sun shining in the sky;
Through teaching how to see one's nature
Such one emerges to abolish all false doctrines.
The Dharma is not inherently sudden or gradual;
Yet according to each person’s delusion,
Enlightenment may come swiftly or slowly.

**Comment by shane0** [comment]
# dogen

mentioned in class


 To study Buddhism is to study the self.
 To study the self is to forget the self.
 To forget the self is to be enlightened by all things.
 To be enlightened by all things is to remove the barriers between oneself and others.


https://www.thezensite.com/ZenTeachings/Dogen_Teachings/GenjoKoan_Aitken.htm

To study the buddha way is to study the self. To study the self is to forget the self. To forget the self is to be actualized by myriad things. When actualized by myriad things, your body and mind as well as the bodies and minds of others drop away. No trace of realization remains, and this no-trace continues endlessly.

**Comment by shane0** [comment]
# Tolerance

- 5 levels restraint recognize realize naturally nirvanic
- story: are you going to dance (elvis & bruce lee)
- story: I'll tell him to forgive you when I see him (guy is upset about some past scenario forgot guy is dead)
- cruel people are the best teachers thank and respect them take the high road when they take the low road
- see all viewpoints use dogen quote learn buddhism & self & forget self & everything enlightens & barrier between dualistic us vs them becomes we unite interconnected arising & empty
